src/sentry/static/sentry/app/views/releaseArtifacts.jsx

```diff
@@ -125,23 +125,27 @@ const ReleaseArtifacts = React.createClass({
         <div className="panel panel-default">
           <div className="panel-heading panel-heading-bold">
             <div className="row">
-              <div className="col-sm-8 col-xs-7">{'Name'}</div>
-              <div className="col-sm-2 col-xs-2">{'Size'}</div>
-              <div className="col-sm-2 col-xs-3 align-right" />
+              <div className="col-lg-7 col-sm-6">{'Name'}</div>
+              <div className="col-lg-2 col-sm-2">{'Distribution'}</div>
```

Comment:
Can we maybe not show this if there isn't a distribution? I see this being complicated and confusion if your project literally has  nothing to do with distributions. Like, most of our users with JavaScript. I feel like seeing "(default)" everywhere will be confusing, and we're not doing anything to explain what it is, etc.

/cc @benvinegar @ckj